modules/tide_content_collection/src/SearchApiIndexHelper.php

```diff
@@ -357,6 +357,18 @@ public function getIndexIntegerFields(IndexInterface $index, array $excludes = [
     return $fields;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getIndexStringFieldPropertyPath(IndexInterface $index, string $field_id) : ?string {
```

Comment:
@joshua-salsadigital I don't see any check to ensure that the field is a string field. Is this method supposed to work with `string` fields only? If not, please rename the method.

Reply:
It can be used for any indexed field. Updated name to generalised. 
`getIndexedFieldPropertyPath`